IGInterfaceDataTable.xcodeproj/project.pbxproj

```diff
@@ -240,6 +240,7 @@
 		29BE4ED81A97A59F0025C38E /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				APPLICATION_EXTENSION_API_ONLY = YES;
```

Comment:
How about we remove this and put up another PR? Just so we can keep PRs localized to a specific change in case we need to revisit a change down the road.